org.scala-ide.sdt.core/src/org/scalaide/core/internal/project/ProjectsDependentSbtBuildManager.scala

```diff
@@ -0,0 +1,39 @@
+package org.scalaide.core.internal.project
+
+import org.scalaide.core.internal.builder.zinc.EclipseSbtBuildManager
+import org.scalaide.core.internal.builder.BuildProblemMarker
+import org.scalaide.ui.internal.preferences.ScalaPluginSettings
+import org.eclipse.core.resources.IFile
+import org.eclipse.core.runtime.SubMonitor
+import org.scalaide.util.internal.SettingConverterUtil
+import scala.tools.nsc.Settings
+import org.scalaide.core.IScalaProject
+import org.eclipse.core.resources.IResource
+import org.scalaide.core.SdtConstants
+import org.scalaide.core.IScalaPlugin
+
+class ProjectsDependentSbtBuildManager(project: IScalaProject, settings0: Settings)
+    extends EclipseSbtBuildManager(project, settings0) {
+  private def areTransitiveDependenciesBuilt = {
+    val projectsInError =
```

Comment:
the name 'projectsInError' is just taken from existing implementation. I'm not sure about changing it. It is maybe a little bit strange construction but rather correct in English though.